securetea/core.py

```diff
@@ -64,16 +70,26 @@ def __init__(self):
             cred['twitter'],
             cred['debug']
         )
+        self.telegram = SecureTeaTelegram(
+            cred['telegram'],
+            cred['debug']
+        )
         if not self.twitter.enabled:
             self.logger.log(
                 "Twitter not configured properly. Exiting...",
                 logtype="error"
             )
             sys.exit(0)
-
+        if not self.telegram.enabled:
+            self.logger.log(
+                "Telegram not configured properly. Exiting...",
+                logtype="error"
+            )
+            sys.exit(0)
         else:
             self.logger.log("Welcome to SecureTea..!! Initializing System")
             self.twitter.notify("Welcome to SecureTea..!! Initializing System")
+            self.telegram.notify("Welcome to SecureTea..!! Initializing System")
```

Comment:
Make it optional, Send a message only if the user has a configuration on his config file.  It will break if the user has not configured it.

Reply:
Right, will update it